README.md

```diff
@@ -79,54 +79,60 @@ Our goals for [v0.29 release cycle][github-release-link] were to focus on adding
 
 Our [prioritized roadmap][roadmap] of features and utilities that the core team is focusing on for the near future. We fixed a lot of localization issues from our initial release but we may not still be perfect. If you find an issue, please file a [localization bug][loc-bug].
 
-#### Highlights from v0.29
+#### Highlights from v0.31
 
 **General**
-- Bug report tool and improved logging.
-- Various localization improvements. 
-- CodeQL added.  Triggered via a cron timer twice a day.
-- "How to use" docs moved to https://docs.microsoft.com/windows/powertoys/
-   - This will allow the community to do direct PRs against those documents
+- Bug report tool and improved logging now added to our system tray.
+- Added in CodeQL and other CI improvements.
+- OOBE Spec should be finalized
 
 **ARM64 Progress**
 - .NET Core upgrade for code bases the PowerToys team controls is complete.  We still have two external dependencies that are .NET Framework that need to be updated.
 
 **Color Picker** 
-- General bug fixes
-- Added ability to provide the name of the color at parity with Office and WinUI Color Picker.
+- Bug fixes in editor
 
 **FancyZones**
-- Allows to use Windows Snap on desktops that don't have a layout applied and for apps that are in the excluded list.
+- Streamlined, simplified user interface 
 - Bug fixes
```

Comment:
I'd add that certain settings (e.g. no. of zones, spacing settings) can now be set on individual layouts. That's pretty major.